beacon-chain/core/helpers/deposits_test.go

```diff
@@ -6,9 +6,16 @@ import (
 
 	"github.com/gogo/protobuf/proto"
 	pb "github.com/prysmaticlabs/prysm/proto/beacon/p2p/v1"
+	"github.com/prysmaticlabs/prysm/shared/featureconfig"
 	"github.com/prysmaticlabs/prysm/shared/params"
 )
 
+func init() {
+	featureconfig.InitFeatureConfig(&featureconfig.FeatureFlagConfig{
```

Comment:
are all these inits required ? isnt it false by default

Reply:
I get panic if i don't init them